jormungandr/src/main.rs

```diff
@@ -217,6 +218,7 @@ fn start_services(bootstrapped_node: BootstrappedNode) -> Result<(), start_up::E
                 fragment_msgbox,
                 enclave,
                 block_msgbox,
+                rewards_report_all,
```

Comment:
Looks like this module begins to want a config struct or something.

Reply:
Well, it does. In one of later patches it starts to trigger a clippy warning.